packages/app/tests/Api/Common-Test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Test file for the common elements of
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2019-01-18
+ */
+
+import React from "react";
+import { createRenderer } from "react-dom/test-utils";
+import tape from "tape";
+import addAssertions from "extend-tape";
+import jsxEquals from "tape-jsx-equals";
+const test = addAssertions(tape, {jsxEquals});
+
+import { concatArray }  from "../../pages/Api/common.js";
+
+test("nullArrayDisplaysNone", (t) => {
```

Comment:
This might seem ridiculous considering the test names themselves are very self-explanatory, but I can't help but feel there should be at least some documentation for what the tests are doing/checking.

Reply:
I think Dr.Dennis likes self descriptive tests.

Reply:
Yeah, that sounds right. Also, considering the amount of tests you would ideally have, needing to document every single one could get exhausting.